### ISSUE TITLE
Error on restore: actor does not exist in organization

### ISSUE THREAD

**Issue by kenmacleod** [chef/knife-ec-backup#83]
Backing up from chef-server 12.3.1, knife-ec-backup-2.0.6, restoring to chef-server 12.9.1, knife-ec-backup 2.0.6, I get the following error on restore:

```
Restoring user ACLs
ERROR: The data in your request was invalid
Response: The actor(s) 'someuser' do not exist in this organization as clients or users.
```

The error is a 400 response from chef server when performing `Initiating PUT to https://cpt-aa010611.foocorp.com/users/someuser/_acl/create` with this data:

```
DEBUG: ---- HTTP Request Body ----
DEBUG: {"create":{"actors":["pivotal","someuser"],"groups":[]}}
```

The target chef server is recently installed and I'm using the following steps leading to the restore:

```
chef-server-ctl cleanse
chef-server-ctl reconfigure
cd $HOME/.chef/trusted_certs
cp /var/opt/opscode/nginx/ca/*.crt .
cd /tmp
rm -rf chef-bkp-20161027/
tar xf /vagrant/chef-bkp-20161027.tgz
/opt/opscode/embedded/bin/knife ec restore /tmp/chef-bkp-20161027 \
  --user pivotal --key /etc/opscode/pivotal.pem \
  --with-user-sql --with-key-sql
```

Before the restore gets to the step "Restoring user ACLs" it does look like all the "Updating user record for ...", "Updating key data for ...", and all the organizations are restored.

Let me know if you want the full logs.

**Comment by poliva83** [comment]
We are also hitting same problem on 12.9.1 but are backup was taken on a 12.9.1 server.

**Comment by jeremiahsnapp** [comment]
This appears to be caused by https://github.com/chef/chef-server/issues/938.

If you have not modified user ACLs on your old server then from what I can tell the simplest solution would be to just avoid backing up and restoring the user ACLs. The creation of the users during the restore process will give them defaults that should work. You can do this by simply appending --skip-useracl to your "knife ec backup/restore" commands.

**Comment by thechile** [comment]
have this as well on server-core 12.13.0, using  `--skip-useracl` allowed the restore to complete but none of the nodes could update chef server. Just received 403 errors.

**Comment by jeremiahsnapp** [comment]
This has been fixed by https://github.com/chef/chef-server/pull/1242 and is available in Chef Server 12.15.3
